### ISSUE TITLE
Forward HTML messages

### ISSUE THREAD

**Issue by mclei** [roundcube/roundcubemail#5818]
I never write HTML messages. And I never reply to HTML messages in HTML.
BUT sometimes I need to forward HTML mail, mainly because of included images.

There are many options in "Compose HTML messages" settings, but this one is missing. Please add this option: "On forward HTML message".

**Comment by alecpl** [comment]
Indeed. You can however use forward as attachment to forward the message as-is.

**Comment by lstandish** [comment]
> I never write HTML messages. And I never reply to HTML messages in HTML.
> BUT sometimes I need to forward HTML mail, mainly because of included images.
> 
> There are many options in "Compose HTML messages" settings, but this one is missing. Please add this option: "On forward HTML message".

I also agree.  I hope the developers will consider adding this one additional compose option.  I never compose in html either, but to be able to _occasionally_ forward_ html I have to have "on forward and reply to html" selected, and manually switch to html in the editor for every message I compose.
